sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1350,7 +1350,12 @@ public Long countCasesForMap(RegionReferenceDto regionRef, DistrictReferenceDto
 						cb.function("date_part", Long.class, cb.parameter(String.class, "date_type"), symptoms2.get(Symptoms.ONSET_DATE)))),
 				new Long(30 * 24 * 60 * 60) // 30 days
 			));
-		Predicate creationDateFilter = cb.lessThan(root.get(Case.CREATION_DATE), root2.get(Case.CREATION_DATE));
+
+		Predicate creationDateFilter = cb.or(
+			cb.lessThan(root.get(Case.CREATION_DATE), root2.get(Case.CREATION_DATE)),
+			cb.or(
+				cb.lessThan(root2.get(Case.CREATION_DATE), DateHelper.getEndOfDay(criteria.getCreationDateFrom())),
```

Comment:
The logic in CaseService.CreateCriteriaFilter is

 		if (caseCriteria.getCreationDateFrom() != null) {
			filter = CriteriaBuilderHelper.and(cb, filter, cb.greaterThan(from.get(Case.CREATION_DATE), DateHelper.getStartOfDay(caseCriteria.getCreationDateFrom())));
		}
		if (caseCriteria.getCreationDateTo() != null) {
			filter = CriteriaBuilderHelper.and(cb, filter, cb.lessThan(from.get(Case.CREATION_DATE), DateHelper.getEndOfDay(caseCriteria.getCreationDateTo())));
		}

So it should by lessThan/greatThanOrEqual, right?